server/routes/api/representatives.js

```diff
@@ -62,9 +93,16 @@ router.get('/:zipCode', async (req, res) => {
         key: CICERO_API_KEY
       },
       paramsSerializer: (params) =>
-        qs.stringify(params, { arrayFormat: 'repeat' })
+        qs.stringify(params, { arrayFormat: 'repeat' }),
+
+      cache: {
+        ttl: 1000 * 60 * 10080 // the time until the cached value is expired in milliseconds: set to 1 week
```

Comment:
Makes the calculation a bit cleaner

```suggestion
        ttl: 1000 * 60 * 60 * 24 * 7 // the time until the cached value is expired in milliseconds: set to 1 week
```